app/src/renderer/vuex/modules/delegation.js

```diff
@@ -159,11 +156,11 @@ export default ({ node }) => {
 
       await dispatch(`sendTx`, {
         type: `postDelegation`,
-        to: wallet.address, // TODO strange syntax
+        to: session.address, // TODO strange syntax
```

Comment:
still strange syntax?

Reply:
yep, just changed the source of the address

Reply:
what's the strange syntax? can we create an issue ?

Reply:
there is already an issue around refactoring the send module #1828